code/game/objects/items/tools/crowbar.dm

```diff
@@ -133,3 +133,30 @@
 	usesound = 'sound/items/jaws_pry.ogg'
 	force = 10
 	toolspeed = 0.5
+	var/list/allowed_tank_containers = list( //This is pretty much just so we avoid attacking every random pipe and whatever on the same tile

```

Comment:
needs dmdoc to meet [documentation requirements](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)